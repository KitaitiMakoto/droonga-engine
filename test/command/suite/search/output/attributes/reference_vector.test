# -*- js -*-
#@require-catalog-version 2
#@disable-logging
{
  "type": "add",
  "dataset": "Default",
  "body": {
    "table": "Memos",
    "key": "Droonga is fun",
    "values": {
      "content": "I started Droonga. It is very fun!",
      "tags": ["Droonga", "Fun"]
    }
  }
}
#@enable-logging
{
  "type": "search",
  "dataset": "Default",
  "body": {
    "queries": {
      "memos": {
        "source": "Memos",
        "output": {
          "elements": ["attributes", "records"],
          "attributes": ["_key", "tags"],
          "limit": 10
        }
      }
    }
  }
}
