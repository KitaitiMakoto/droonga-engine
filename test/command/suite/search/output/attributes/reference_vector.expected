{
  "inReplyTo": "request-id",
  "statusCode": 200,
  "type": "search.result",
  "body": {
    "memos": {
      "attributes": [
        {
          "name": "_key",
          "type": "ShortText",
          "vector": false
        },
        {
          "name": "tags",
          "type": "Tags",
          "vector": true
        }
      ],
      "records": [
        [
          "Droonga is fun",
          [
            "Droonga",
            "Fun"
          ]
        ]
      ]
    }
  }
}
